includes/functions-compatibility.php

```diff
@@ -129,40 +129,45 @@ function dokan_get_product_downloads( $product ) {
  * @param string $date_to
  */
 function dokan_save_product_price( $product_id, $regular_price, $sale_price = '', $date_from = '', $date_to = '' ) {
-    $product_id    = absint( $product_id );
+    $product = wc_get_product( absint( $product_id ) );
+
```

Comment:
Consider removing this empty line and aligning the previous line (132) as per below lines ( from 134 to 137).